### PR TITLE
Feature/add limit to the keypress track in alphanumeric input

### LINKED ISSUE #41: ✨ Add limit to the keypress track in AlphanumericInput
The users should not be able to generate more than **70** events (type more than 70 times). After reaching this limit, the experience is ignored.

AlphanumericInput should have an `onCancel` prop for it.

Next thing to do
- [ ] alert the user that the experiment is ignored under the chunk
- [ ] chunk border red in the input
- [ ] inform the user that he can not type more than 70 characters for technical reason on the home page

### PR DESCRIPTION
**Pull request by johannchopin**
## Description

Close #41.

Check the Familiarization page and type 70x time.

## Checklist

- [x] I compare my branch with `develop`
- [x] All tests passed after running `yarn s-check && yarn lint`

### CODE REVIEW

File: src/global.scss [hunk 1]
```diff
@@ -19,3 +19,21 @@ html, body {
...
+
+.form-control {
+  &.success {
+    border-color: $success;
+
+    &:focus {
+      box-shadow: rgba($success, .25) 0px 0px 0px 0.25rem;
+    }
+  }
+
```
Comment (Lyokolux) [inline src/global.scss:31]: Is this class useless now ?

File: src/pages/FamiliarizationPage.svelte [hunk 2]
```diff
@@ -21,4 +21,9 @@
   bind:events
   chunkLength={CHUNK_LENGTH}
   onSuccess={() => { $swiper.slideNext() }}
+  onCancel={() => {
+    setTimeout(() => {
+      $swiper.slideNext()
+    }, 1500)
+  }}
```
Comment (Lyokolux) [inline src/pages/FamiliarizationPage.svelte:28]: May be refactorised into a function ?

**Review by Lyokolux — CHANGES_REQUESTED** [review]

File: src/pages/FamiliarizationPage.svelte (hunk 2 above)
Comment (johannchopin) [reply src/pages/FamiliarizationPage.svelte:28]: This is just 💩

File: src/global.scss (hunk 1 above)
Comment (johannchopin) [reply src/global.scss]: ✅

**Pull request merged**